packages/ag-charts-enterprise/src/features/annotations/annotationsModule.ts

```diff
@@ -26,6 +26,13 @@ export const AnnotationsModule: _ModuleSupport.Module = {
                 handle: {
                     fill: _Theme.DEFAULT_ANNOTATION_HANDLE_FILL,
                 },
+                axisLabel: {
+                    enabled: true,
+                    color: 'white',
+                    fill: _Theme.DEFAULT_ANNOTATION_STROKE,
```

Comment:
Probably one for when we iterate with design team, but we'll likely need two new theme variables here for axis label color and fill.